### PR TITLE
Captain Agent PR part 2.

### PR DESCRIPTION
**Pull request by LeoLjl**
Part 2 of CaptainAgent PR. It implements the code specified in the blog post introduced in #27. The full context in #21.

## Checks

- [ ] I've included any doc changes needed for https://ag2ai.github.io/autogen/. See https://ag2ai.github.io/ag2/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by qingyun-wu** [comment]
Thanks @LeoLjl! Reviewing it. Could you fix the doc failure first?

**Comment by qingyun-wu** [comment]
Please fix docs failure

File: notebook/test_captainagent.ipynb [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+      "\n",
+      "\n",
+      "--------------------------------------------------------------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = user_proxy.initiate_chat(\n",
+    "    captain_agent,\n",
+    "    message='Assemble a team of experts, find the stock price of Microsoft in the past 1 year and plot a line chart to show the trend. Save the line chart as \"microsoft_stock_price.png\".',\n",
```
[the remark this answers is not in the document]
Comment (LeoLjl) [reply notebook/test_captainagent.ipynb:441]: will remove it in the next commit.

**Review by qingyun-wu — APPROVED** [review]
Looks good to me now! Thanks Jiale! I think we are ready to move on to Part 3.

**Comment by qingyun-wu** [comment]
> Looks good to me now! Thanks Jiale! I think we are ready to move on to Part 3.

@skzhang1 In the meanwhile, could you make a PR to update the visualization or upload it here? Thank you!

**Pull request merged**